canonical-web-and-design/vanilla-framework
# Update navigation to use new tagged logo

## Done

- Added new tagged logo styles as default
- Added a deprecated file for the old light and dark defaults
- Bump version

NOTE: We still don't have the logos for LXD or Vanilla so I've just used the circle of friends for now

Fixes #4375 

## QA

- Open [demo](https://vanilla-framework-4380.demos.haus/docs/examples)
- Look through all the navigation examples and check the logo looks ok
- Check the deprecated file - not sure if the light and dark theme are ok in the same file?

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)


## Screenshots

![Screenshot 2022-03-28 at 16 56 06](https://user-images.githubusercontent.com/58959073/160438498-f4c3ad07-e4eb-4e15-8a63-b71f75113cbe.png)
